aospy/test/test_objs/models.py

```diff
@@ -0,0 +1,19 @@
+from aospy.model import Model
+import runs
+
+
+am2 = Model(
```

Comment:
Let me know if 973d868 addresses this to your liking.  It's not as general as we might like (since the test doesn't just look for the grid files, it also looks for the olr files), but I think it is sufficient to determine whether a test is only possible to be run in the GFDL environment.